packages/plans-grid-next/src/components/actions.tsx

```diff
@@ -256,10 +191,10 @@ const LoggedInPlansFeatureActionButton = ( {
 		isFreePlan( planSlug ) ||
 		( storageAddOnsForPlan && ! canPurchaseStorageAddOns && nonDefaultStorageOptionSelected )
 	) {
-		if ( planActionOverrides?.loggedInFreePlan ) {
+		if ( action?.loggedInFreePlan ) {
 			return (
 				<PlanButton planSlug={ planSlug } onClick={ onCtaClick } current={ current }>
-					{ planActionOverrides.loggedInFreePlan.text }
+					{ action.loggedInFreePlan.text }
 				</PlanButton>
```

Comment:
> Part of this refactor, I'd expect that we go deep on restructuring this file to no longer need overrides like this. From what I'm seeing so far, we haven't actually removed PlanActionOverrides - we just moved it from a prop to something that's passed through the useAction hook instead, which to me feels like the same thing. The idea is to no longer "override" anything, which by principle will lead to the type of cleanup we need here (at least to my mind).

Hey @chriskmnds we're on the same page :)

I should've left more detailed notes, but what you're describing is actually my current plan. You'll notice I've removed overrides already from the `LaunchPagePlanFeatureActionButton` and the `SignupFlowPlanFeatureActionButton`.

I stopped at `LoggedInPlansFeatureActionButton`, which is where we still see remnants of the PlanActionOverrides,  because I wanted to get a confidence check on the direction of the PR. Also, the button, by far, has the most logic to lift into the `useAction` hook 😅 

Rest assured, all references to PlanActionOverrides are slated to be removed by the end of this PR. It sounds like, generally though, things are moving in the right direction, so I'll do my best to finish the refactoring asap.